### ISSUE TITLE
Library is logging its own calls to Zipkin

### ISSUE THREAD

**Issue by JamesNK** [open-telemetry/opentelemetry-dotnet#108]
I took your TestApp.AspNetCore.2.0 and changed the exporter to Zipkin.

```cs
services.AddSingleton<ZipkinTraceExporter>((p) =>
{
    var exportComponent = p.GetService<IExportComponent>();

    return new ZipkinTraceExporter(
      new ZipkinTraceExporterOptions()
      {
          Endpoint = new Uri("http://localhost:9411/api/v2/spans"),
          ServiceName = typeof(Program).Assembly.GetName().Name,
      },
      exportComponent);
});
```

As soon as the app starts running it sends constant requests to Zipkin, and it is logging those requests. Within seconds there are thousands of traces.

![image](https://user-images.githubusercontent.com/303201/59487014-68d9f600-8ecf-11e9-9b70-e40c1bfe6171.png)

**Comment by SergeyKanzhelev** [comment]
there is a custom sampler you can implement to filter out unwanted requests: https://github.com/open-telemetry/opentelemetry-dotnet/blob/aec6feb6f4c63d5a5fab13b0c9f33b60eaffaf33/src/OpenTelemetry.Collector.Dependencies/DependenciesCollectorOptions.cs#L44 

But clearly there should be a better way. Ideally out of the box.

Similar issue: https://github.com/open-telemetry/opentelemetry-dotnet/issues/69

**Comment by lmolkova** [comment]
Should be fixed with #167 and #259
